### DISCUSSION TITLE
ScriptReference/Compilation.AssemblyBuilder-additionalReferences

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#420] [Notes]
# ScriptReference/Compilation.AssemblyBuilder-additionalReferences

https://docs.unity3d.com/ScriptReference/Compilation.AssemblyBuilder-additionalReferences.html

**Comment by fydar** [comment]
To get this to work with Assembly Definition Files you need to enter the path of where Unity builds the DLLs relative to the project folder. Below is an example.  
`assemblyBuilder.additionalReferences = new string[] { "Library/ScriptAssemblies/MyAssembly.dll" };`
